app/utils/http_callbacks.py

```
@@ -1,15 +1,53 @@
-    except Exception as e:
-        logging.error("HTTP callback error: %s", e)
 
+    data = {"event": event_type, "payload": payload}
+    headers = headers or {}
+    attempt = 0
+    while True:
+        try:
+            requests.post(url, json=data, timeout=timeout, headers=headers)
+            logging.info("HTTP callback sent to %s", url)
+            break
+        except Exception as exc:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)